org.eclipse.scanning.api/src/org/eclipse/scanning/api/scan/event/IPositionListener.java

```diff
@@ -51,29 +52,41 @@ default boolean positionWillPerform(PositionEvent evt) throws ScanningException
 	 *   <em>Note:</em> throwing an exception will stop the scan. If this behaviour is
 	 *   not desirable the exception should be caught and logged instead 
 	 */
-	default void levelPerformed(PositionEvent evt) throws ScanningException {
+	default void levelPerformed(PositionEvent event) throws ScanningException {
 		// default implementation does nothing, subclasses should override as necessary
 	}
 	
 	/**
-	 * Called when the position changes.
+	 * Called when the position of an {@link IScannable} changes.
 	 * @param event
 	 * @throws ScanningException if an exception occurred responding to this event.
 	 *   <em>Note:</em> throwing an exception will stop the scan. If this behaviour is
 	 *   not desirable the exception should be caught and logged instead 
 	 */
-	default void positionChanged(PositionEvent evt) throws ScanningException {
+	default void positionChanged(PositionEvent event) throws ScanningException {
 		// default implementation does nothing, subclasses should override as necessary
 	}
 
 	/**
-	 * Called after a given position is reached.
+	 * Called after a given position in a scan has been performed.
 	 * @param event
 	 * @throws ScanningException if an exception occurred responding to this event.
 	 *   <em>Note:</em> throwing an exception will stop the scan. If this behaviour is
 	 *   not desirable the exception should be caught and logged instead 
 	 */
-	default void positionPerformed(PositionEvent evt) throws ScanningException {
+	default void positionPerformed(PositionEvent event) throws ScanningException {
 		// default implementation does nothing, subclasses should override as necessary
 	}
+	
+	/**
+	 * Called after a given position in a scan has been moved to but before
+	 * detectors are exposed.
+	 * TODO: is there a better name for this method
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)